src/oemof/solph/_facades.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+"""
+Solph Facade based on oemof.network.SubNetwork
+
+SPDX-FileCopyrightText: Uwe Krien <krien@uni-bremen.de>
+SPDX-FileCopyrightText: Pierre-François Duc
+
+SPDX-License-Identifier: MIT
+
+"""
+
+from oemof.network import SubNetwork
+
+
+class Facade(SubNetwork):
+    # attributes
+    def __init__(self,*args,**kwargs):
+        self.facade_type = None
+
+    def add_subnode(self,*args):
+        """Add a subnode to the subnetwork"""
+        for sub_component in args:
+            sub_component.label = self._sub_component_labelling(sub_component.label)
```

Comment:
Apparently cannot set label once it is set for renaming components in subnetwork
 - let labels be overwritten in oemof-solph
 - workaround with custom_properties "sub_label"

Reply:
Labels are meant to be unique and constant identifiers at the level of `oemof.network`. Why do the labels have to be changed?

Reply:
Because we wanted to label the components within a subnetwork in a specific way, but we found a solution, we do not change the labels, instead we use a method to label the subnetwork components right upon creation